jwcrypto/jwk.py

```diff
@@ -1016,9 +1017,8 @@ def import_from_pem(self, data, password=None, kid=None):
                     raise e
 
         self.import_from_pyca(key)
-        if kid is None:
```

Comment:
This change breaks the current behavior of import_from_pem() when a kid is not provided explicitly

Reply:
If the kid is not provided it will be set to the thumbprint in the call to import_from_pyca, which is the same as the old behaviour, no?

Reply:
Oh, you are right!